front/login.php

```diff
@@ -93,5 +93,6 @@
         'errors'    => $auth->getErrors(),
         'login_url' => $CFG_GLPI["root_doc"] . '/front/logout.php?noAUTO=1' . str_replace("?", "&", $REDIRECT),
     ]);
+    http_response_code(401);
```

Comment:
Is this working?

HTTP code is sent in headers, but `TemplateRenderer::getInstance()->display()` call would already have send output if PHP output buffering is not activated.

Reply:
It works for me.

Without the PR I see a 200 code response when an invalid username or password is used.

With the PR, I now see a 401.